apps/dashboard/app/models/batch_connect/app.rb

```diff
@@ -387,7 +399,7 @@ def add_cluster_widget(attributes, attribute_list)
         else
           attributes[:cluster] = {
             value: clusters.first.id.to_s,
-            widget: "hidden_field"
+            fixed: true
```

Comment:
This is not really a part of this - I just noticed that it should be fixed instead of hidden.

Reply:
Jeff why should this be fixed instead of `hidden_field`?  I have been scratching my head for weeks as to why the dynamic forms do not work in a single-cluster case, and it comes down to this change.  when the cluster is fixed, the form does not have a cluster field, and as such none of the `data-option-for-cluster-*` options work at all.  This makes it very hard to share apps between clusters.  

You can see my discord post here:   https://discourse.openondemand.org/t/issues-with-dynamic-form/3663

I dug into this issue for a while and found this change, and swapping it back to `hidded_field`, suddenly all the cluster options start working.

Reply:
>  This makes it very hard to share apps between clusters.

I suppose I didn't account for this case. I'll open a bug ticket and revert it.